src/pages/Shop.tsx

```diff
@@ -0,0 +1,52 @@
+import Card from "../components/Card/Card";
+import Header from "../components/Header";
+import Drawer from "../components/Drawer/Drawer";
+import {useState, useEffect} from "react";
+
+type Item = {
```

Comment:
add export to be able to import it

```suggestion
exort type Item = {
```

Reply:
fixed